### PR TITLE
Convert IP and MAC addresses into different formats

### PR DESCRIPTION
**Pull request by vasu-dasari**

### CODE REVIEW

File: rebar.config [hunk 1]
```diff
@@ -1,2 +1,16 @@
-{deps, []}.
 {erl_opts, [debug_info,warn_export_vars,warn_shadow_vars,warn_obsolete_guard]}.
+
+{deps, [
```
Comment (benoitc) [inline rebar.config]: why this dependency?

**Review by benoitc — CHANGES_REQUESTED** [review]
can you remove the `sync` dependency? or maybe make it part of a dev profile if needed. But I don't think it's needed in prod. let me know :)

**Comment by benoitc** [comment]
i can also just make the change myself in a different commit. Just let me know what's your preference. Otherwise the patch looks good.

File: rebar.config (hunk 1 above)
Comment (vasu-dasari) [reply rebar.config]: Adding this dependecy makes it easy during development. Basically it scans for the files that have been modified and compiles those files and then hot-patches them on your running Erlang VM. That is a left over suring my development/debugging phase that should not have been included in a library like `inet_ext`. Will remove the dependency.

**Review by benoitc — APPROVED** [review]

**Comment by benoitc** [comment]
thanks! merged

**Comment by vasu-dasari** [comment]
> thanks! merged

Thanks @benoitc

**Pull request merged**